docs/deploy-development-rp.md

```diff
@@ -269,6 +269,12 @@ After that, when you [create](https://github.com/Azure/ARO-RP/blob/master/docs/d
   curl -X PATCH -k "https://localhost:8443/admin/subscriptions/$AZURE_SUBSCRIPTION_ID/resourceGroups/$RESOURCEGROUP/providers/Microsoft.RedHatOpenShift/openShiftClusters/$CLUSTER/etcdrecovery" 
   ```
 
+* Delete a managed resource
+  ```bash
+  RESOURCEID=<id of managed resource to delete>
```

Comment:
nit: consider naming this `MANAGED_RESOURCEID` so it is not confused with the resourceID which is part of the url / request